modules/ROOT/pages/common-builder-customizations.adoc

```diff
@@ -99,6 +99,53 @@ By default, Experience Cloud generates the URL for your portal domain. You can c
 The new domain URL displays in the *Your chosen new My Domain URl* field.
 . Click *Save*.
 
+== Configure Feed Component
```

Comment:
Configure the Feed Component
or 
Configure Feed Components

Reply:
Configure the Feed Component